network-api/networkapi/wagtailpages/models.py

```diff
@@ -1276,3 +1277,27 @@ def serve(self, request):
         # Note that due to how this page type works, there is no
         # associated template file in the wagtailpages directory.
         return HttpResponseRedirect(self.URL)
+
+
+class SpecialTemplatePage(FoundationMetadataPageMixin, Page):
+    ALLOWED_TEMPLATES = [
+        ('unspecified', '-'),
+        ('wagtailpages/one-off-pages/youtube.html', 'Youtube Campaign'),
+    ]
```

Comment:
I expect us to want to tightly control this list, rather than giving users the freedom to "just type a template".